src/store/Reducers/index.js

```diff
@@ -0,0 +1,35 @@
+const initialState  = {
+    additionalPrice: 0,
+    car: {
+      price: 26395,
+      name: '2019 Ford Mustang',
+      image:
+        'https://cdn.motor1.com/images/mgl/0AN2V/s1/2019-ford-mustang-bullitt.jpg',
+      features: []
+    },
+    store: [
+      { id: 1, name: 'V-6 engine', price: 1500 },
+      { id: 2, name: 'Racing detail package', price: 1500 },
+      { id: 3, name: 'Premium sound system', price: 500 },
+      { id: 4, name: 'Rear spoiler', price: 250 }
+    ]
+} ;
+
+export function reducer(state = initialState , action) {
+   switch(action.type){
+       case "ADD" :
```

Comment:
Nice job getting the logic. I would suggest you also remove/filter out the feature from the store when you call ADD, and put it back in the store when you call REMOVE. This will prevent the user from buying more than 1 of the same feature.